hack/test-cmd.sh

```diff
@@ -785,12 +804,18 @@ __EOF__
     kube::test::get_object_assert 'services mock' "{{${labels_field}.status}}" 'replaced'
     kube::test::get_object_assert 'rc mock' "{{${labels_field}.status}}" 'replaced'
     # Command
-    kubectl label -f $file labeled=true "${kube_flags[@]}"
+    # We need to set --overwrite, because otherwise, if the first attempt to run "kubectl label" 
```

Comment:
I'm not sure that I agree with this logic.  It seems that your call to kubectl-with-retry needs to succeed, or this test should bail out as failed?  You should not need (or want) --overwrite here?